src/ui/components/RadioButton.stories.tsx

```diff
@@ -1,10 +1,10 @@
 import { ComponentStory, ComponentMeta } from '@storybook/react'
-import React from 'react'
+import React, { useState } from 'react'
 
 import { RadioButton } from './RadioButton'
 
 export default {
-  title: 'ui/RadioButton',
+  title: 'ui/buttons/RadioButton',
```

Comment:
je pense que ça devrait plutot etre

```suggestion
  title: 'ui/inputs/RadioButton',
```